sdk/textanalytics/ai-text-analytics/src/generated/models/index.ts

```diff
@@ -90,6 +90,14 @@ export interface Entity {
    * Entity sub type, such as Age/Year/TimeRange etc
    */
   subCategory?: string;
+  /**
+   * Start position for the entity text.
```

Comment:
encoding needed here too

Reply:
If this was already added and this is not the place for the comments, please ignore my comment.